source/module_hamilt_lcao/module_gint/gint_k_pvdpr.cpp

```diff
@@ -39,28 +45,32 @@ void Gint_k::allocate_pvdpR(void)
 
 void Gint_k::destroy_pvdpR(void)
 {
-    ModuleBase::TITLE("Gint_k","destroy_pvpR");
+    ModuleBase::TITLE("Gint_k", "destroy_pvpR");
 
     const int nspin = GlobalV::NSPIN;
-    assert(nspin>0);
-    
-	for(int is =0;is<nspin;is++) 
-	{
-		delete[] pvdpRx_reduced[is];
-	}
+    assert(nspin > 0);
+
+#pragma omp parallel for
+    for (int is = 0; is < nspin; is++)
+    {
+        delete[] pvdpRx_reduced[is];
+    }
     delete[] pvdpRx_reduced;
 
-	for(int is =0;is<nspin;is++) 
-	{
-		delete[] pvdpRy_reduced[is];
-	}
+#pragma omp parallel for
```

Comment:
this part is not time consuming, so we don't need openmp

Reply:
但是加上应该也没坏处

Reply:
nspin should be a small number, and the extra overhead introduced by OpenMP here will reduce performance. And it will obviously make the code confusing.